server/services/interventionsService.ts

```diff
@@ -210,6 +211,20 @@ export default class InterventionsService {
     })) as DraftReferral
   }
 
+  async amendNeedsAndRequirements(
+    token: string,
+    referralId: string,
+    needsAndRequirements: Partial<ReferralNeedsAndRequirement>
+  ): Promise<ReferralNeedsAndRequirementUpdate> {
+    const restClient = this.createRestClient(token)
+
+    return (await restClient.patch({
```

Comment:
David asked us to switch the other endpoints to post, best double check if this should remain as patch